.github/workflows/security-scanning.yml

```
@@ -0,0 +1,401 @@
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Kube-score
+        run: |
+          curl -L https://github.com/zegl/kube-score/releases/latest/download/kube-score_linux_amd64 -o kube-score
+          chmod +x kube-score
+
+      - name: Scan K8s manifests with Kube-score
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2162:info:1:41: read without -r will mangle backslashes

(shellcheck)